launch/sick_safetyscanners_nodelet.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<launch>
+
+    <arg name="nodelet_manager_name" default="sick_nodelet"/>
+    <arg name="robot_name" default="$(optenv ROBOT robot)"/>
+    <arg name="laser_name" default="laser"/>
+
+    <group>
+        <!-- Sick SafetyScanners nodelet -->
+        <rosparam file="$(find sick_safetyscanners)/launch/config/sick_safetyscanners.yaml" command="load" subst_value="true"/>
+
+        <!-- Nodelet manager -->
+        <node pkg="locus_nodelet_manager" type="locus_nodelet_manager" name="$(arg nodelet_manager_name)"
+            respawn="true" respawn_delay="$(optenv LOCUS_RESPAWN_DELAY 10)" output="screen">
+        </node>
+    </group>
```

Comment:
Maybe give an option to launch the nodelet manager or not? Maybe we want to launch these nodelets with an existing nodelet manager by just passing the `nodelet_manager_name`.